src/components/transaction-item.tsx

```diff
@@ -188,21 +187,33 @@ const LargeVersion = React.memo(
         </Flex>
         {!hideRightElements ? (
           <Stack alignItems="flex-end" textAlign="right" as="span" spacing="tight">
+            <Timestamp tx={tx} />
             <Flex justifyContent="flex-end" alignItems="flex-end" flexWrap="wrap">
               {tx.tx_status === 'pending' ? (
-                <Caption fontWeight="bold" as="span" color={color('feedback-alert')}>
+                <Caption mr="6px" as="span">
                   Pending
                 </Caption>
               ) : null}
-
               {tx.tx_status !== 'pending' && tx.tx_status !== 'success' ? (
-                <Caption fontWeight="bold" mr="tight" as="span" color={color('feedback-error')}>
+                <Caption mr="6px" as="span" color={color('feedback-error')}>
                   Failed
                 </Caption>
               ) : null}
-              <Timestamp tx={tx} />
+              {tx.tx_status === 'success' && !!tx.microblock_hash ? (
```

Comment:
I used tx.is_unanchored to detect microblocks, I think it's better because the tx will still have the attribute microblock_hash even when it is anchored
Example https://stacks-node-api-microblocks.testnet.stacks.co/extended/v1/address/STM6S3AESTK9NAYE3Z7RS00T11ER8JJCDNTKG711/transactions?limit=50

Reply:
It will have it but be an empty string (which is working), but I agree with you now that we have `is_unanchored` to work with. Good call. 👍

Reply:
I do not see the `is_unanchored` is ever `true`, do you? I don't think this is working?